### PR TITLE
[MIG][13.0] sale_elaboration: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT21470
@Tardo @carlosdauden review

### CODE REVIEW

File: sale_elaboration/models/sale_order.py [hunk 1]
```diff
@@ -49,24 +49,38 @@ class SaleOrderLine(models.Model):
...
+
+    @api.depends("elaboration_id")
+    def _compute_elaboration_note(self):
+        for line in self:
+            line.elaboration_note = line.elaboration_id.name
 
-    @api.onchange("elaboration_id")
-    def onchange_elaboration_id(self):
-        self.elaboration_note = self.elaboration_id.name
+    def _inverse_elaboration_note(self):
```
Comment (carlosdauden) [inline sale_elaboration/models/sale_order.py]: Remove unused method

File: sale_elaboration/models/sale_order.py [hunk 2]
```diff
@@ -49,24 +49,38 @@ class SaleOrderLine(models.Model):
...
-    def product_id_change(self):
-        result = super().product_id_change()
-        self.is_elaboration = self.product_id.is_elaboration
-        return result
+    @api.depends("product_id")
+    def _compute_is_elaboration(self):
+        for line in self:
+            line.is_elaboration = line.product_id.is_elaboration
+
+    def _inverse_is_elaboration(self):
```
Comment (carlosdauden) [inline sale_elaboration/models/sale_order.py]: Remove unused method

File: sale_elaboration/tests/test_sale_elaboration.py [hunk 3]
```diff
@@ -94,7 +94,7 @@ def test_search_elaboration(self):
 
     def test_sale_elaboration_change(self):
         self.order.order_line.elaboration_id = self.elaboration_b.id
-        self.order.order_line.onchange_elaboration_id()
+        # self.order.order_line.onchange_elaboration_id()
```
Comment (carlosdauden) [inline sale_elaboration/tests/test_sale_elaboration.py]: Remove

**Review by carlosdauden — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
@carlosdauden  Changes done

File: sale_elaboration/tests/test_sale_elaboration.py (hunk 3 above)
Comment (carlosdauden) [reply sale_elaboration/tests/test_sale_elaboration.py]: And this? :smile:

**Review by carlosdauden — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@carlosdauden now yes!!!!! :smile:

File: sale_elaboration/views/sale_elaboration_report_views.xml [hunk 4]
```diff
@@ -9,7 +9,7 @@
         <field name="model">sale.order.line</field>
         <field name="arch" type="xml">
             <tree string="Elaborations" create="false">
-                <field name="confirmation_date"/>
+                <field name="date_order"/>
```
Comment (pedrobaeza) [inline sale_elaboration/views/sale_elaboration_report_views.xml:12]: Why are you using order date now instead of the confirmation date?

Comment (sergio-teruel) [reply sale_elaboration/views/sale_elaboration_report_views.xml:12]: Un V13 the confirmation_date order field is deleted, now confirmation date is date_order if order is confirmed

**Pull request merged**